### ISSUE TITLE
F g27

### ISSUE THREAD

**Issue by Hexality** [joshcancode/wheelie-for-logitech-g27#1]
wish this had at least working code 
windows 11 officially killed the wheel now, can't even install LGS Wingman without it breaking core isolation/integrity.

**Comment by joshcancode** [comment]
> wish this had at least working code windows 11 officially killed the wheel now, can't even install LGS Wingman without it breaking core isolation/integrity.

Yeah sorry. I was originally developing this as a web application using Electron, but I no longer have the wheel. The program was able to process input, but I never got further than basic inputs and customizable wheel rotation. If you'd like, I can send you a .zip of where I left off.

**Comment by Hexality** [comment]
actually it would be perfect, the wheel rotation is exactly the main problem I'm having to deal with (its locked to 180 degree)

**Comment by joshcancode** [comment]
> actually it would be perfect, the wheel rotation is exactly the main problem I'm having to deal with (its locked to 180 degree) [bounce.webm](https://user-images.githubusercontent.com/17398632/187090349-019204aa-2013-4bff-b7c0-e382ecd24446.webm)

Excellent video, quite silly and marvelous. Can you add me on Discord, so I can send you the project? Josh_#9838
